### ISSUE TITLE
GMs can't make characters using this

### ISSUE THREAD

**Issue by Eaglefire0** [RabidOwlbear/osr-character-builder#18]
I was wondering if you could add the ability for GMs to create characters with this

**Comment by RabidOwlbear** [comment]
Do you mean GM's creating character classes?

**Comment by Eaglefire0** [comment]
No, I meant creating NPCs using this. I know I can create the random retainers as the GM, but I'd like if possible to have the same creation that the players have.
